src/metabase/driver/generic_sql.clj

```diff
@@ -85,6 +86,9 @@
     "Return a korma form appropriate for converting a Unix timestamp integer field or value to an proper SQL `Timestamp`.
      SECONDS-OR-MILLISECONDS refers to the resolution of the int in question and with be either `:seconds` or `:milliseconds`."))
 
+(extend-protocol jdbc/IResultSetReadColumn
```

Comment:
I think, extending the protocol has effect on ALL jdbc drivers. Could not find another way, though, to correctly deal with the `Object[]` object from the jdbc result set.
This will display Crate's [`ARRAY` type](https://crate.io/docs/reference/sql/data_types.html#array) correctly, if it is a "root column".
`ARRAY` inside an `OBJECT` column is still displayed as `[Ljava.langObject@...` - I think the `Object` type from the database needs to be converted recursively.